src/API/Notification.js

```diff
@@ -0,0 +1,7 @@
+import { notification } from 'antd';
+export const openNotification = (message,description) => {
```

Comment:
```suggestion
export const openNotification = (message, description='') => {
```
тогда тут https://github.com/perfomance-review/pr-frontend/blob/b4336fbb3f35b0a42501aa2d3d52eb347adb4097/src/pages/Polls.jsx#L80
будет просто openNotification('Результаты опроса будут доступны после deadline')